src/pages/editor/EditorInstitution.tsx

```diff
@@ -91,6 +113,36 @@ export const EditorInstitution = () => {
             {customer?.rboInstitution ? t('editor.institution.rbo_funded') : t('editor.institution.not_rbo_funded')}
           </Typography>
 
+          {institutionUsers && (
+            <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '3rem' }}>
+              {institutionAdmins && (
+                <div>
```

Comment:
```suggestion
                <>
```

Reply:
Må ha gyldig `html-tag` for at `flex` skal fungere ;)

Reply:
TIL